### DISCUSSION TITLE
Add Not-Starred-Filter Option for Trends

### DISCUSSION THREAD

**Discussion by Amejonah1200** [community/community#14899] [Other Feature Feedback, Questions, & Ideas]
As I use the Trends Tab mainly for starring interesting new projects. Over time, and by starring projects which are like in trends everyday, I start to find it difficult to scroll and star new ones as I'm being "spammed" with already starred/viewed projects, I would be happy to have those filtered out to be able to view directly unstarred ones.
(category can also be "general" or "Feed")

**Comment by Tr1985** [comment]
> As I use the Trends Tab mainly for starring interesting new projects. Over time, and by starring projects which are like in trends everyday, I start to find it difficult to scroll and star new ones as I'm beeing "spammed" with already starred/viewed projects, I would be happy to have those filtered out to be able to view directly unstarred ones.

**Reply by Amejonah1200** [reply]
_\*trying to understand the purpose of this quotation .\_.\*_
